backend/scripts/monitor_ai_processing.py

```
@@ -0,0 +1,338 @@
+        print(f"      ID: {content_item.id}")
+        print(f"      状态: {content_item.processing_status}")
+        print(f"      Summary: {self._get_status_emoji(status_summary)} {status_summary}")
+        print(f"      Key Points: {self._get_status_emoji(status_key_points)} {status_key_points}")
+        print(f"      Metadata: {self._get_status_emoji(status_metadata)} {status_metadata}")
+        
+        # 如果有失败，显示详细信息
+        if status_summary == "failed" or status_key_points == "failed":
+            if content_item.error_message:
+                print(f"      错误: {content_item.error_message}")
+            if ai_result and not ai_result.summary:
+                print(f"      Summary问题: 数据库中无summary数据")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)